src/components/frame/RouteLayouts/UserNavigation/UserNavigation.tsx

```diff
@@ -23,16 +25,26 @@ const MSG = defineMessages({
     id: `${displayName}.userReputationTooltip`,
     defaultMessage: 'This is your share of the reputation in this colony',
   },
+  walletAutologin: {
+    id: `${displayName}.walletAutologin`,
+    defaultMessage: `{isMobile, select,
+      true {Connecting...}
+      other {Connecting wallet...}
+    }`,
+  },
 });
 
+const { query700: query } = queries;
+
 const UserNavigation = () => {
   const { colony } = useColonyContext();
-
   const { wallet } = useAppContext();
   const { formatMessage } = useIntl();
+  const isMobile = useMediaQuery({ query });
```

Comment:
It seems like we're following the same pattern in multiple files:
- Import `queries`
- Destructure the `query700`
- Pass it to `useMediaQuery` to get `isMobile`

Maybe it's worth refactoring it into its own hook?

Reply:
Sounds good, added a hook for this `useMobile`